### PR TITLE
add DARK & light theme

### PR DESCRIPTION
**Pull request by Starship-Headlight**

### CODE REVIEW

File: themes/dark-and-light/v1/theme.json [hunk 1]
```diff
@@ -0,0 +1,151 @@
+{
+  "$schema": "https://raw.githubusercontent.com/streetwriters/notesnook-themes/main/schemas/v1.schema.json",
+  "name": "DARK & light",
```
Comment (thecodrr) [inline themes/dark-and-light/v1/theme.json:3]: Is this casing intended?

File: themes/dark-and-light/v1/theme.json [hunk 2]
```diff
@@ -0,0 +1,151 @@
...
+  "license": "GPL-3.0-or-later",
+  "homepage": "https://notesnook.com",
+  "description": "Dark theme with light everything ⛔ONLY FOR WEB/DESKTOP APP. MAY HAVE SOME ISSUES IN SMARTPHONE APP⛔",
+  "colorScheme": "dark",
+  "compatibilityVersion": 1,
+  "authors": [
+    {
+      "name": "MONARCH",
+      "email": "N/A",
+      "url": "N/A"
```
Comment (thecodrr) [inline themes/dark-and-light/v1/theme.json:15]: You can omit these.

**Comment by thecodrr** [comment]
You don't have to close and open a new PR everytime you want to make changes. Just make the necessary changes and commit them.

File: themes/dark-and-light/v1/theme.json (hunk 1 above)
Comment (Starship-Headlight) [reply themes/dark-and-light/v1/theme.json:3]: Yes. At least till the time, I find a better name for it😁

File: themes/dark-and-light/v1/theme.json (hunk 2 above)
Comment (Starship-Headlight) [reply themes/dark-and-light/v1/theme.json]: Yep. I did it. Could you please check it again.

**Comment by Starship-Headlight** [comment]
@thecodrr  could you please publish the themes. I updated them just now.

**Pull request merged**